### ISSUE TITLE
EME: Protected/unprotected images need empty alts

### ISSUE THREAD

**Issue by dstorey** [MicrosoftEdge/Demos-old#325]
https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/eme/

The text proceeds the padlock icons, so they should have alt="" role="none"

We may want to change the "protected" text to DRM protected, to give the user (especially screen reader users that can't use the visual clues) more context in what that means, rather than just reading "tears of steel, protected"
